model/model_training/configs/config.yaml

```diff
@@ -29,7 +29,7 @@ defaults:
     - cnn_dailymail
     - multi_news
     - scitldr
-    - soda
+    # - soda  # TODO: fix list index out of range error
```

Comment:
still to do?

Reply:
that's still an open todo, just checked it. Will create a seperate PR to fix this.